Last-Update: 2015-06-14
Forwarded: not-needed
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: disable xt_ECHO module.
 Disable xt_ECHO module on Linux 3.x to fix DKMS compilation on 4.x.
 This is to prevent xt_ECHO from appearing in DKMS configuration template
 generated on 3.x. from the list of built modules.
 Upstream don't build xt_ECHO on kernels v4+.

--- a/extensions/Kbuild
+++ b/extensions/Kbuild
@@ -9,11 +9,11 @@
 obj-${build_CHAOS}       += xt_CHAOS.o
 obj-${build_DELUDE}      += xt_DELUDE.o
 obj-${build_DHCPMAC}     += xt_DHCPMAC.o
 obj-${build_DNETMAP}     += xt_DNETMAP.o
-ifeq (${VERSION},3)
-obj-${build_ECHO}        += xt_ECHO.o
-endif
+#ifeq (${VERSION},3)
+#obj-${build_ECHO}        += xt_ECHO.o
+#endif
 obj-${build_IPMARK}      += xt_IPMARK.o
 obj-${build_LOGMARK}     += xt_LOGMARK.o
 obj-${build_SYSRQ}       += xt_SYSRQ.o
 obj-${build_TARPIT}      += xt_TARPIT.o
